README.md

```diff
@@ -48,7 +48,7 @@ with:
 
 ```yaml
 with:
-    json-file-path: 'your-filename.json'
+  json-file-path: "your-filename.json"
```

Comment:
line 33 uses single quotes, so why does this one (and others) change to double quotes? Consistency is important